core/templates/pages/email-dashboard-pages/email-dashboard-page.component.html

```diff
@@ -1,26 +1,34 @@
-<div class="oppia-query-form">
-  <form class="form" ng-submit="$ctrl.submitQueryAsync()">
-    <label for="label-target-not-logged-n-days">Has not logged in for n days: </label>
-    <input id="label-target-not-logged-n-days" type="number" min="0" ng-model="$ctrl.hasNotLoggedInForNDays">
-    <br>
-    <label for="label-target-inactive-for-ndays">Inactive in last n days:</label>
-    <input id="label-target-inactive-for-ndays" type="number" min="0" ng-model="$ctrl.inactiveInLastNDays">
-    <br>
-    <label for="label-target-created-more-n-exps">Has created at least n explorations:</label>
-    <input id="label-target-created-more-n-exps" type="number" min="0" ng-model="$ctrl.createdAtLeastNExps">
-    <br>
-    <label for="label-target-created-fewer-n-exps">Has created fewer than n explorations:</label>
-    <input id="label-target-created-fewer-n-exps" type="number" min="0" ng-model="$ctrl.createdFewerThanNExps">
-    <br>
-    <label for="label-target-edited-more-n-exps">Has edited at least n explorations:</label>
-    <input id="label-target-edited-more-n-exps" type="number" min="0" ng-model="$ctrl.editedAtLeastNExps">
-    <br>
-    <label for="label-target-edited-fewer-n-exps">Has edited fewer than n explorations: </label>
-    <input id="label-target-edited-fewer-n-exps" type="number" min="0" ng-model="$ctrl.editedFewerThanNExps">
-    <br>
-    <input type="submit" value="submit query">
-    <input type="reset" value="reset" ng-click="$ctrl.resetForm()">
-  </form>
+<div>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-10">
+        <md-card class="oppia-page-card oppia-long-text">
+          <form class="form-horizontal" ng-submit="$ctrl.submitQueryAsync()" name="form.schemaForm">
+            <legend>Email Dashboard</legend>
+            <div ng-repeat="customizationArgSpec in $ctrl.customizationArgSpecs track by $index">
+              <div class="d-xl-inline-flex form-group">
+                <label class="col-3 col-form-label" for="label-target-role-method">
+                  <[customizationArgSpec.description]>:
+                </label>
+                <div class="col-7">
+                  <schema-based-editor local-value="$ctrl.data[$index].value"
+                                       schema="customizationArgSpec.schema"
+                                       not-required="$ctrl.notRequired"
```

Comment:
can't you put `true` here instead of `$ctrl.notRequired`?

Reply:
Done.